### DISCUSSION TITLE
Welcome to asleep Discussions!

### DISCUSSION THREAD

**Discussion by angerhang** [OxWearables/asleep#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZxR62666** [comment]
Do I need to buy the UKBB dataset?

**Reply by aidendoherty** [reply]
There is no need to do this. The asleep tool can be used on your own in-house dataset.
